configure.acr

```diff
@@ -43,16 +43,16 @@ IFAND HAVE_LIB_Z USE_ZIP {
 		LIBZIP = -lz -lzip ;
 	}{
 		USE_LIB_ZIP = 0 ;
-		LIBZIP = -lrz ; (( static linking to shlr/zip ))
+		LIBZIP = ; (( static linking to shlr/zip ))
```

Comment:
if you are removing this and its building it means that there's no way to build against system zlib+libz?

Reply:
No, building against system zlib+libz works as before. See the `shlr/zip/deps.mk`:
```make
ifeq ($(USE_LIB_ZIP),1)
LINK+=$(LIBZIP)
else
LINK+=$(SHLR)/zip/librz.$(EXT_AR)
CFLAGS+=-I$(SHLR)/zip/include
endif
```